### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by Ziyang1992** [fengjian0106/hed-tutorial-for-document-scanning#6]
The order of "logits" and "targets" in sentences "cost = tf.nn.weighted_cross_entropy_with_logits(logits=logits, targets=label, pos_weight=pos_weight)" may be wrong in python 3.6.

**Comment by fengjian0106** [comment]
@Ziyang1992 hi, 你有遇到python报错吗，python2.7是没有这种问题的

**Comment by Ziyang1992** [comment]
Python没有报错，只是结果会有问题,交叉熵会出现负值。这可能和tensorflow版本所提供的API有关。我的是tensorflow1.6.0

**Comment by fengjian0106** [comment]
logits=logits, targets=label  这种写法是语法允许的
